polina.vasileva/task-2-1/cmd/service/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import "fmt"
+
+const (
+	MINTEMP = 15
```

Comment:
<img width="400" height="468" alt="Image" src="https://github.com/user-attachments/assets/84f1d06c-2374-4b52-9eda-ac70d09e4907" />

```suggestion
	minTemp = 15
```